public/app/features/search/components/ManageDashboards.tsx

```diff
@@ -153,7 +153,9 @@ const getStyles = stylesFactory((theme: GrafanaTheme) => {
       display: flex;
       flex-direction: column;
       flex: 1;
-      margin-top: ${theme.spacing.xl};
+      height: 100%;
+      padding-top: ${theme.spacing.lg};
+      margin-top: -${theme.spacing.md};
```

Comment:
It seems the overflow behaviour still exists when you view it on a smaller screen. I would suggest you add more margin below...`margin-top: -${theme.spacing.xl}` .

Reply:
yeah...